x-pack/plugins/fleet/common/openapi/components/schemas/output_elasticsearch.yaml

```diff
@@ -0,0 +1,59 @@
+title: elasticsearch
+type: object
+properties:
+  id:
+    type: string
+  is_default:
+    type: boolean
+  is_default_monitoring:
+    type: boolean
+  name:
+    type: string
+  type:
+    type: string
+    enum: ['elasticsearch']
+  hosts:
+    type: array
+    items:
+      type: string
+  ca_sha256:
+    type: string
+  ca_trusted_fingerprint:
+    type: string
+  config:
+    type: object
+  config_yaml:
+    type: string
+  ssl:
+    type: object
+    properties:
+      certificate_authorities:
+        type: array
+        items:
+          type: string
+      certificate:
+        type: string
+      key:
+        type: string
+  proxy_id:
+    type: string
+  shipper:
+    type: object
+    properties:
+      disk_queue_enabled:
+        type: boolean
+      disk_queue_path:
+        type: string
+      disk_queue_max_size:
+        type: number
+      disk_queue_encryption_enabled:
+        type: boolean
+      disk_queue_compression_enabled:
+        type: boolean
+      compression_level:
+        type: number
+      loadbalance:
+        type: boolean
+required:
```

Comment:
I think hosts should probably be required here too

Reply:
Will be taken care of with separate POST / PUT schemas.

Reply:
Addressed in a74dc5815f26c1d0fd9051f699cba32ad18ba8ad